src/Entity/IdeasWorkshop/Note.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\EntityIdentityTrait;
+use AppBundle\Entity\EntityNameSlugTrait;
+use AppBundle\Entity\EntityTimestampableTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ *
+ * @ORM\Table(
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="note_slug_unique", columns="slug")
+ *     }
+ * )
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Note
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+    use EntityNameSlugTrait;
+
+    private const PUBLISHED_INTERVAL = 'P3W';
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Theme")
+     */
+    private $theme;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Scale")
+     */
+    private $scale;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="Need")
+     */
+    private $needs;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Adherent")
+     */
+    private $adherent;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $publishedAt;
+
+    /**
+     * @var Committee
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Committee")
+     */
+    private $committee;
+
+    /**
+     * @var ArrayCollection
+     *
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\IdeasWorkshop\Guideline")
+     */
+    private $guidelines;
+
+    /**
+     * @ORM\Column(length=11, options={"default": "IN_PROGRESS"})
```

Comment:
- What about adding an `Assert\Choice` with a callback to check the status ?
- Can you use, for the default option, the `IN_PROGRESS` const of the `NoteStatusEnum` class please ?